roles/monitoring-client/tasks/client.yml

```diff
@@ -0,0 +1,47 @@
+---
+
+- name: Ensure monitoring_client directory exists
+  file:
+    state: directory
+    path: "/etc/docker-compose/{{ client_data.name|default('monitoring_client') }}"
```

Comment:
recurse: yes?

Reply:
as its likely /etc/docker-compose doesn't exist at this point

Reply:
from the manual `If directory, all immediate subdirectories will be created if they do not exist, since 1.7 ` recurse: yes is for permissions

See http://docs.ansible.com/ansible/latest/file_module.html